config/database.yml

```diff
@@ -2,6 +2,8 @@ login: &login
   adapter: postgresql
   host: localhost
   port: 5432
+  username: postgres
+  password: 
```

Comment:
If you need this on your local system that's fine.  But please don't check it in... other open source users won't be expecting you to change the DB config.